Antidote/DialingCallViewController.m

```diff
@@ -61,10 +63,30 @@ - (void)createEndCallButton
     [self.view addSubview:self.cancelCallButton];
 }
 
+- (void)createFriendAvatar
+{
+    AvatarsManager *avatars = [[AvatarsManager alloc] init];
+
+    CGFloat diameter = (1.0 / 2.0) * self.view.bounds.size.width;
```

Comment:
This will be broken in landscape. I think you can just define some constant for diameter, that would be enought.